json-files/Michael

```diff
@@ -0,0 +1,11 @@
+​{
+"name": "Michael Demone",
+"email": "michael.demone@mail.utoronto.ca",
+"github": "mademone",
+"techskills": 3,
+"periods": ["ancient", "medieval", "early modern", "modern", "contemporary"],
+"regions": ["Europe", "North America"],
+"styles": ["political", "diplomatic", "cultural", "social", "art", "intellectual", "economic"],
+"picture": "../images/your-image.jpg",
+"blurb": "../markdown-files/your-markdown-file.md"
```

Comment:
change this to reflect the actual name of hte markdown file you wrote -- which right now is `../markdown-files/"Michael` .  You should however rename this file to `Michael.md` if you are able to do so -- and if you can, then adjust this value accordingly.

Reply:
I have no idea how to make any of the changes you have suggested.

Reply:
> I have no idea how to make any of the changes you have suggested.

I think I have now completed this.